nni/compression/pytorch/compressor.py

```diff
@@ -668,15 +670,20 @@ def forward(ctx, tensor, quant_type, wrapper, **kwargs):
         else:
             raise ValueError("unrecognized QuantType.")
 
-        bits = QuantGrad.get_bits_length(wrapper.config, quant_type)
-        qmin, qmax = torch.Tensor([0], device=tensor.device), torch.Tensor([(1 << bits) - 1], device=tensor.device)
-        ctx.save_for_backward(tensor, wrapper.module.scale, wrapper.module.zero_point, qmin, qmax)
+        bits = QuantGrad.get_bits_length(wrapper.config, QType_Dict[quant_type])
+        qmin, qmax = torch.Tensor([0]).to(tensor.device), torch.Tensor([(1 << bits) - 1]).to(tensor.device)
```

Comment:
How to deal with the case of signed integer?

Reply:
I think it is another thing which is not related to this pr. May be we will discuss it in the future release.